src/index.js

```diff
@@ -183,9 +183,11 @@ Object.assign(experimental, {
 //
 
 import {default as ReflectionEffect} from './core/experimental/reflection-effect/reflection-effect';
+import {default as GPUScreenGridLayer} from './core-layers/experimental/screen-grid-layer/screen-grid-layer';
```

Comment:
Nit: Should we rename the files to gpu-screen-grid-layer and move to experimental-layers?

Reply:
Then add to layer browser?